src/pages/community-partners.astro

```diff
@@ -48,13 +48,28 @@ const partners = [
 ---
 
 <Layout metaTitle="Community Partners | Dev Protocol">
-<div class="mt-28 lg:mt-20 max-w-full text-center">
-  <h2 class="mx-auto w-full font-display text-5xl m-8 dark:text-white">Community Partners</h2>
-  <h4 class="mx-auto w-full font-display text-xl m-4 p-8 dark:text-gray-100">Community Partners help us educate people about web3 and, become a part of their creator success</h4>
+<div class="flex flex-col md:gap-y-4 gap-y-8 mt-28 max-w-full text-center">
+  <h2 class="mx-auto w-full font-display text-5xl dark:text-white">Community Partners</h2>
+  <h4 class="mx-auto w-full font-display text-xl px-8 md:p-0 dark:text-gray-100">Community Partners help us educate people about web3 and, become a part of their creator success</h4>
+  <a href="#">
```

Comment:
Actually, now that you mention the link, @Simperfy, I was thinking to make a form for community partnerships. Google Forms would be good (or forms directly on our website). @web3community/dev-protocol-devx, @web3community/core-maintainer, thoughts? I could work on this as well. 🙂

Reply:
I think gforms is bit uncomfortable, plus this should be asked from the devprtcl team not us 😅 @aggre

Reply:
We can use choose one of these if ever https://tailwindui.com/components/marketing/sections/contact-sections

Reply:
Looks like we need some feedback from dev protocol team, let me know if anything comes up.

Reply:
I'm now asking aggre and mayu. Please wait for the response.

Reply:
https://tailwindui.com/components/marketing/sections/contact-sections#component-dec976a631662b509173ba8b848c49cd we decided to choose this.

The fields information will be 

Community Name
Website
Email
Why do you want to be partner with us?
How did you hear about us?

Reply:
I think add that feature first before we merge this, so we can direct them there if they click **Become a Community Partner** ?

Reply:
> https://tailwindui.com/components/marketing/sections/contact-sections#component-dec976a631662b509173ba8b848c49cd we decided to choose this.
> 
> The fields information will be
> 
> Community Name Website Email Why do you want to be partner with us? How did you hear about us?

Where would we store this information? Do we have a database for this?

Reply:
> I think add that feature first before we merge this, so we can direct them there if they click **Become a Community Partner** ?

Yeah sure.